### ISSUE TITLE
Incorrect Docker .camply location

### ISSUE THREAD

**Issue by grantland** [juftin/camply#169]
**Describe the bug**

Docs show `.camply` location for Docker at `/home/camply/.camply` but right now it seems to be reading it from `/root/.camply`:

* https://github.com/juftin/camply/blob/main/docs/docker.md#running-in-docker
* https://github.com/juftin/camply/blob/main/docs/examples/docker-compose.yaml

**Original Camply Command (with `--debug`)**

```console
docker run \
  --name camply-file-share-example \
  --env TZ="America/Denver" \
  --volume ${HOME}/.camply:/home/camply/.camply \
  juftin/camply \
  camply campsites \
      --provider yellowstone \
      --start-date 2023-07-22 \
      --end-date 2023-07-27 \
      --continuous \
      --notifications email
```

**Expected behavior**
Should run with configuration from `${HOME}/.camply`

**Actual behavior**

Configuration from `${HOME}/.camply` are not used

**Additional context**

I checked an instance and when using `camply configure` from inside the Docker instance it creates it to `/root/.camply` and references it from there

**Comment by juftin** [comment]
Thanks for reporting @grantland. Looks like I messed that up on the release of camply `0.7.0` https://github.com/juftin/camply/pull/135/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557- let me get that addressed

**Comment by juftin** [comment]
Actually - my Docker CI/CD isn't currently being triggered like I expected. This is close to being fixed

**Comment by juftin** [comment]
Okay - this is *actually* resolved now on [camply 0.7.6](https://github.com/juftin/camply/releases/tag/v0.7.6) - the `.camply` file will be expacted @ `/home/camply/.camply`
